app/javascript/app/components/emission-pathways-scenario-table/emission-pathways-scenario-table-selectors.js

```diff
@@ -17,6 +18,28 @@ const getIndicatorsData = createSelector(getIdData, data => {
   return data.indicators || null;
 });
 
+export const getCategories = createSelector(getIndicatorsData, data => {
+  if (!data) return null;
+  return ['All']
+    .concat(uniq(data.map(i => i.category)))
+    .map(c => ({ value: c, label: c }));
+});
+
+const getSelectedCategory = createSelector(
```

Comment:
I am not sure I understand the purpose of this format. Why getCategories when all you are doing is finding out the category from the url. And then in the getSelectedCategoryObject, you are doing unecessary work to find it again. I think it should be:
get category from url -> filter categories with value for getCategories and return the object. 

If you are getting the categories array and mapping them correctly with the All option in there too there is no need. 

Additionally, you dont need the 'All' option at all. You can just set the selector to allow the user to clear it like in the rest of the site. We have done this many times in other components.